src/client/backup/reset/reset.tsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react'
 import styles from './reset.scss'
+import { Container } from '../../common/Container'
+import {clear} from '../../configuration/LocalRepository'
 import {PrimaryButton} from '../../common/forms/Button'
 import {iBin} from '../../common/fonts/Icons'
-import {Messages, MessagesType} from '../../common/Messages'
+
 export function Reset() {
-    const messageType = MessagesType.ERROR
-    const messages = [ 'Reset your Nevergreen configuration back to defaults. Please note, resetting your configuration can not be undone!' ]
+    
     const resetConfiguration = () => {
-        window.localStorage.clear()
+        clear()
         window.location.reload()
     }
     return(
-        <div>
-        <Messages type={messageType}
-                messages={messages}
-                data-locator='messages'/>
-        <PrimaryButton className={styles.import}
+        <>
```

Comment:
You don't need the Fragment `<>` here, the `Container` can be the top level element